content/blog/2019/02/2019-02-08-let-s-celebrate-java-11-support.adoc

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title: "Let's celebrate Java 11 Support on Jenkins"
+tags:
+- core
+- developer
+- java11
+- community
+- platform-sig
+author: alecharp
+links:
+  gitter: /jenkinsci/platform-sig
+  googlegroup: jenkins-platform-sig
+  sig: platform
+---
+
+NOTE: This is a joint blogpost prepared by the Java 11 Support Team: Adrien Lecharpentier, Baptiste Mathus, Kevin Earl, Mark Waite, Ramon Leon, Oleg Nenashev, Maria Isabel Vilacides.
```

Comment:
Earls. :-)

Oleg Nenashev, and Maria Isabell Vilacides

Reply:
I'm so sorry...

```suggestion
NOTE: This is a joint blogpost prepared by the Java 11 Support Team: Adrien Lecharpentier, Baptiste Mathus, Kevin Earls, Mark Waite, Ramon Leon, Oleg Nenashev, and Maria Isabel Vilacides.
```